blocks/commerce-cart/commerce-cart.js

```diff
@@ -149,8 +155,37 @@ export default async function decorate(block) {
   // Events
   events.on(
     'cart/data',
-    (payload) => {
-      toggleEmptyCart(isCartEmpty(payload));
+    (cartData) => {
+      const urlParams = new URLSearchParams(window.location.search);
+      const itemUid = urlParams.get('itemUid');
+
+      if (itemUid && cartData?.items) {
+        const itemExists = cartData.items.some((item) => item.uid === itemUid);
+        if (itemExists) {
+          const updatedItem = cartData.items.find((item) => item.uid === itemUid);
+          const productName = updatedItem.name || updatedItem.product?.name || 'Product';
+          const message = (placeholders?.Cart?.UpdatedProductMessage || '{product} was updated in your shopping cart.')
```

Comment:
Since these labels/placeholders were added in the content source. Once double check they work as expected with the test URL, we could remove these hard-coded strings. 
```suggestion
          const productName = updatedItem.name || updatedItem.product?.name
          || placeholders?.Cart?.UpdatedProductName;
          const message = placeholders?.Cart?.UpdatedProductMessage?.replace('{product}', productName);
```